### ISSUE TITLE
Cannot read properties of null (reading 'getVideoTracks')

### ISSUE THREAD

**Issue by Fnopster** [aws/amazon-chime-sdk-js#2659]
### What happened and what did you expect to happen?

We've received many `Cannot read properties of null (reading 'getVideoTracks')` errors in Sentry.

It was thrown by this code snippet:
```
audioVideoDidStart() { 
 this.mediaStreamBroker.mediaStream.getVideoTracks().length>0&&this.contentAudioVideo.videoTileController.startLocalV...
```

This appears to be from the [DefaultContentShareController](https://github.com/aws/amazon-chime-sdk-js/blob/fac13706eca8f559c8e5d292028a524923aff2b1/src/contentsharecontroller/DefaultContentShareController.ts#L160).

Looking at the code, it appears the ContentShareMediaStreamBroker can set the media stream to null when it cleans up. 

Should the DefaultContentShareController check if mediaStream exists before invoking getVideoTracks?

### Have you reviewed our existing documentation?

- [X] GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] FAQs [page](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-js#amazon-chime-sdk-for-javascript)
- [X] Feature-specific [guides](https://github.com/aws/amazon-chime-sdk-js#guides)

### Reproduction steps

No repo steps at this time. Have only seen this error in Sentry.

### Amazon Chime SDK for JavaScript version

3.10.0

### What browsers are you seeing the problem on?

Chrome

### Browser version

Chrome 111.0

### Browser console logs

N/A

**Comment by ltrung** [comment]
That code is triggered when it started not when it cleans up. This seems like a dup of this issue https://github.com/aws/amazon-chime-sdk-js/issues/2535. 
Do you have any audio sharing use case where there is no video track?

**Comment by kennyvallejodev** [comment]
also, the `mediaStreamBroker` has the `mediaStream` return type as always defined (`get mediaStream(): MediaStream`) when actually is initialized as undefined and has a setter that is used when you do the `startContentShare`. 

so why not checking for `mediaStream` to actually be defined? i actually mentioned this before on my issue #2535

**Comment by kennyvallejodev** [comment]
@ltrung 

I was doing some research in the chime SDK and for this error to happen seems to be a weird race condition where either `audioVideoDidStop` or `stopContentShare` is executed first than the `audioVideoDidStart` because if what you said before on https://github.com/aws/amazon-chime-sdk-js/issues/2535#issuecomment-1397809558 is true about checking on `startContentShare` that the mediaStream exists before is assigned. 

having in mind that we use the `DefaultMeetingSession` that has all this things by default, we never touch or do any cleanup on the `mediaStream`.

so even if the fix is easy to fix (safe guard that mediaStream exists on `audioVideoDidStart` (this is the only place where we are not checking this inside the `DefaultContentShareController.ts`) so will propose a PR to fix this issue!

**Comment by dylonChime** [comment]
This bug has been fixed in version v3.16.0. Please reopen or submit a support case if this is still an issue.
